src/carousel/carousel.scss

```diff
@@ -0,0 +1,193 @@
+  overflow-x: overlay;
+  counter-reset: item;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+  -webkit-overflow-scrolling: touch;
+  scrollbar-color: transparent transparent;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+
+  &::-webkit-scrollbar {
+    width: 0;
+    height: 0;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-color: transparent;
+    border: none;
+
+    &:hover {
+      background-color: transparent;
+    }
+  }
+
+  &-item {
+    position: relative;
+    flex-shrink: 0;
+    background-position: center;
+    background-size: cover;
+    counter-increment: item;
+    scroll-snap-align: center;
+    width: 100%;
+    box-sizing: border-box;
+    @supports (aspect-ratio: 16 / 9) {
+      aspect-ratio: 16 / 9;
+    }
+
+    &-title {
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      padding: $iui-baseline $iui-sm;
+      box-sizing: border-box;
+      width: 100%;
+      @include themed {
+        background-color: rgba(0, 0, 0, t(iui-opacity-4));
+        color: t(iui-color-foreground-accessory);
+      }
+      @media (prefers-reduced-motion: no-preference) {
+        transition: background-color $iui-speed-fast ease;
+      }
+
+      &:hover {
+        @include themed {
+          background-color: rgba(0, 0, 0, t(iui-opacity-3));
+        }
+      }
+    }
+  }
+}
+
+@mixin iui-carousel-navigation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: $iui-baseline * 3;
+
+  &-container {
+    display: flex;
+    flex-grow: 1;
+    white-space: nowrap;
+    overflow: hidden;
+    justify-content: center;
+  }
+
+  &-dots {
+    @include iui-reset;
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &-dot {
+    @include iui-reset;
+    width: $iui-baseline * 2;
+    height: $iui-baseline * 2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @at-root {
+      a#{&} {
+        &:hover:not(.iui-invisible):not(.iui-active)::after {
+          @include themed {
+            background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-3));
+          }
+        }
+
+        &.iui-invisible {
+          cursor: default;
+        }
+
+        &:focus {
+          outline: none;
+
+          &::after {
+            @include themed {
+              outline: 1px solid t(iui-color-foreground-primary);
+              outline-offset: 1px;
+            }
+          }
+        }
+
+        &:focus:not(:focus-visible)::after {
+          @include themed {
+            outline: none;
+          }
+        }
+      }
+    }
+
+    &::after {
+      content: '';
+      display: block;
+      width: $iui-icons-small;
+      height: $iui-icons-small;
+      border-radius: 50%;
+      border: 1px solid transparent;
+      @media (prefers-reduced-motion: no-preference) {
+        transition: background-color $iui-speed-fast ease, border-color $iui-speed-fast ease, width $iui-speed-fast ease,
```

Comment:
Where do you change border-color?

Reply:
I think that may be leftover from my initial attempt at building this.  I have removed border-color from the transition.